### ISSUE TITLE
Feature suggestion: Switch apps via Vimac

### ISSUE THREAD

**Issue by blackketter** [dexterleng/vimac#293]
I find myself wanting to use Vimac to click on windows belonging to apps that are not frontmost, in order to switch to another app.  

Would it be possible to put a single shortcut in each visible window to allow clicking between applications?  

This would be quicker and more intuitive than switching via the command-tab switcher or other name-based (Spotlight, Alfred, LaunchBar) switching apps and would allow immediate selection of specific windows.

**Comment by lougreenwood** [comment]
Nice idea!

**Comment by dexterleng** [comment]
@blackketter @lougreenwood @mmroczka have yall given Slate a shot? I just found out Slate supports window hints.

**Comment by blackketter** [comment]
I just gave [Slate](https://github.com/jigish/slate) a try.  Yep, it does provide hints for visible windows.  

Unfortunately it appears to have been abandoned 8 years ago, unfinished.  The good news is its open source and hopefully some of the window hinting code can be of use for Vimac!

**Comment by mmroczka** [comment]
Yeah, I used to use Slate, but abandoned it after I saw the dev stopped working on it. :(
